src/internal/internal_he2hb_gemm.cc

```diff
@@ -67,9 +69,11 @@ void he2hb_gemm(
         #pragma omp task slate_omp_default_none \
             shared( A, B, C ) \
             firstprivate( alpha, beta, panel_rank, i, layoutc, call_tile_tick ) \
+            firstprivate( one, zero ) \
```

Comment:
Why adding these two `const scalar_t` here?

Reply:
`default(none)` still requires that they get specified, even though they're constant.